portfolio/src/main/webapp/comments.js

```diff
@@ -0,0 +1,55 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Creates and returns a list element
+ * @param {string} text
+ * @return {!Element}
+ */
+function createListElement(text) {
+  const liElement = document.createElement('li');
+  liElement.textContent = text;
+  return liElement;
+}
+
+/**
+ * Sends a request to a servlet that deletes all the comments, then refetches the comments 
+ * to delete them from the page
+ */
+function deleteComments() {
+    fetch('/delete-comments')
+    .then(displayComments());
```

Comment:
must have missed this before, but do we have to call `displayComments()` after deleting them?

Reply:
I think so because I need the dom to be refreshed. The button doesn't remove the comments on the front-end, so calling displayComments again ensures that if someone had added a comment right after someone deleted all the comments, the new comment is displayed and if there are really no comments, then the dom is updated by removing the old comments and displaying no comments.

Reply:
Got it, thanks for the explanation!